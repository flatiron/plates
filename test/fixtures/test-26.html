<div class="authors">
  <ul>
    <li class="author">
      <h2 class="title"></h2>
      <a class="name" href="">The author name is<span class="inner"></span></a>
    </li>
  </ul>
</div>