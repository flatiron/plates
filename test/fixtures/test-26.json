{
  "author":
    { "name": {"inner": "Author 1"}, "title": "Title 1", "link": "//1" }
}