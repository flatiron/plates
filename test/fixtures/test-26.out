<div class="authors">
  <ul>
    <li class="author">
      <h2 class="title">Title 1</h2>
      <a class="name" href="//1">The author name is<span class="inner">Author 1</span></a>
    </li>
  </ul>
</div>